src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -3,33 +3,74 @@ package no.nav.familie.ef.mottak.service
 import no.nav.familie.ef.mottak.config.DOKUMENTTYPE_SKJEMA_ARBEIDSSØKER
 import no.nav.familie.ef.mottak.integration.IntegrasjonerClient
 import no.nav.familie.ef.mottak.repository.domain.Soknad
+import no.nav.familie.kontrakter.felles.journalpost.Journalstatus
 import no.nav.familie.kontrakter.felles.oppgave.*
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 import java.time.LocalDate
 
 @Service
 class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
                      private val søknadService: SøknadService) {
 
-    fun lagJournalføringsoppgave(søknadId: String) {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+
+    fun lagJournalføringsoppgave(søknadId: String): Long? {
         val soknad: Soknad = søknadService.get(søknadId)
         if (soknad.dokumenttype == DOKUMENTTYPE_SKJEMA_ARBEIDSSØKER) {
             val aktørId = integrasjonerClient.hentAktørId(soknad.fnr)
 
             val journalpostId: String = soknad.journalpostId ?: error("Søknad mangler journalpostId")
-            val opprettOppgave = OpprettOppgave(
-                    ident = OppgaveIdent(ident = aktørId, type = IdentType.Aktør),
-                    saksId = null,
-                    journalpostId = journalpostId,
-                    tema = Tema.ENF,
-                    oppgavetype = Oppgavetype.Journalføring,
-                    fristFerdigstillelse = LocalDate.now(),
-                    beskrivelse = "Enslig mor eller far som er arbeidssøker (15-08.01)",
-                    behandlingstema = null,
-                    enhetsnummer = null
-            )
-
-            integrasjonerClient.lagOppgave(opprettOppgave)
+
+            val journalpost = integrasjonerClient.hentJournalpost(journalpostId)
+            if (journalpost.journalstatus == Journalstatus.MOTTATT) {
+                return when {
+                    finnOppgaver(journalpost.journalpostId, Oppgavetype.Journalføring).antallTreffTotalt > 0L -> {
+                        log.info("Skipper oppretting av journalførings-oppgave. " +
+                                 "Fant åpen oppgave av type ${Oppgavetype.Journalføring} for ${journalpost.journalpostId}")
+                        null
+                    }
+                    finnOppgaver(journalpost.journalpostId, Oppgavetype.Fordeling).antallTreffTotalt > 0L -> {
+                        log.info("Skipper oppretting av journalførings-oppgave. " +
+                                 "Fant åpen oppgave av type ${Oppgavetype.Fordeling} for ${journalpost.journalpostId}")
+                        null
+                    }
+                    else -> {
+                        val nyOppgave = lagJournalføringsoppgave(journalpostId, aktørId)
+                        log.info("Oppretter ny journalførings-oppgave med id ${nyOppgave.oppgaveId} " +
```

Comment:
`oppgaveId =${nyOppgave.oppgaveId}` så er det enklere å søke hvis det trengs, samme sak gjelder rad 31, 36, og 47

Reply:
Her skjønte jeg ikke hva du mente.

Reply:
Når dette logges i loggene så er det enklere hvis man bruker `key=value` for felter som eks id'er
Då blir de automatiskt indeksert og enklere å søke etter i kibana